framework/arcane-framework/src/test/scala/services/storage/models/amazon/AmazonS3StoragePath.scala

```diff
@@ -0,0 +1,56 @@
+import com.sneaksanddata.arcane.framework.services.storage.models.amazon.AmazonS3StoragePath
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.prop.TableDrivenPropertyChecks.forAll
+import org.scalatest.prop.Tables.Table
+
+class AmazonS3StoragePath extends AnyFlatSpec {
+
+  "AmazonS3StoragePath" should "be able to parse correct path" in {
+    val path = "s3a://bucket/key"
+    val parsed = AmazonS3StoragePath.fromHdfsPath(path)
+    assert(parsed.isSuccess)
```

Comment:
these are .net/java style asserts. I'd suggest we use [matchers](https://www.scalatest.org/user_guide/using_matchers) that look like `result should be 1`
Please review https://stackoverflow.com/questions/61528075/scalatest-combine-assertions
and this one ofc https://www.scalatest.org/user_guide/using_matchers

Reply:
Fixed